tasks/proposals/gigadot-update.ts

```diff
@@ -0,0 +1,119 @@
+// @ts-nocheck
+import {
+  ConfigNames,
+  getReserveAddress,
+  loadPoolConfig,
+} from "../../helpers/market-config-helpers";
+import {
+  generateProposal,
+  getApi,
+  location,
+  generateProposalV2,
+  dispatchAs,
+  rootEvmCall,
+  padAddress,
+  account,
+} from "../../helpers/hydration-proposal.js";
+import { MARKET_NAME } from "../../helpers/env";
+import { task } from "hardhat/config";
+import {
+  addTransaction,
+  getBatch,
+  clearBatch,
+} from "../../helpers/transaction-batch";
+import {
+  FORK,
+  getACLManager,
+  getPoolAddressesProvider,
+  getPoolConfiguratorProxy,
+  POOL_ADMIN,
+  ZERO_ADDRESS,
+} from "../../helpers";
+import { network } from "hardhat";
+import ProposalDecoder from "../../helpers/proposal-decoder";
+import { exit } from "process";
+import { getPotRewardsStrategy } from "../../helpers/contract-getters";
+import chalk from "chalk";
+
+task(`gigadot-update`, ``).setAction(async function (_, hre) {
+  const { utils } = hre.ethers;
+  const config = await loadPoolConfig(MARKET_NAME as ConfigNames);
+  const { poolAdmin } = await hre.getNamedAccounts();
+  const signer = await hre.ethers.getSigner(poolAdmin);
+  const poolConfigurator = (await getPoolConfiguratorProxy()).connect(signer);
+  const poolAddressesProvider = await getPoolAddressesProvider();
+  const aclManager = (
+    await getACLManager(await poolAddressesProvider.getACLManager())
+  ).connect(signer);
+  console.log("poolAdmin", poolAdmin);
+  const networkId = FORK ? FORK : hre.network.name;
+  const admin = POOL_ADMIN[networkId];
+  const isPoolAdmin = await aclManager.isPoolAdmin(admin);
+  const hydrationTx = (await getApi()).tx;
+
+  const txs = [];
+
+  if (!isPoolAdmin) {
+    console.error("not pool admin " + admin);
+    return;
+  }
+
+  console.log("review and update supply caps");
+  await hre.run("review-supply-caps", { fix: true, batch: true });
+  for await (const el of getBatch()) {
```

Comment:
why is this necessary?

Reply:
proposal generated should be called from dispatcher aave admin anyway, not as root

Reply:
https://apidocs.bsx.fi/Hydration/pallets/Dispatcher/extrinsic/dispatch_as_aave_manager

Reply:
It generate same call as `update-supply-caps` tasks. It's same as https://github.com/galacticcouncil/aave-v3-deploy/blob/aa75553e48b95ded80a75f93a7b1823cdd4cea89/helpers/hydration-proposal.js#L127-L132 it's just not hidden under `generateProposal()`. 
It's dispatched as `admin`